Symfony/CS/Fixer/PSR2/EofEndingFixer.php

```diff
@@ -25,13 +26,25 @@ public function fix(\SplFileInfo $file, $content)
     {
         // [Structure] A file must always end with a linefeed character
 
-        $content = rtrim($content);
+        if (empty($content)) {
+            return $content;
+        }
+
+        $tokens = Tokens::fromCode($content);
+        $count = $tokens->count();
+        if (0 === $count) {
+            return;
```

Comment:
BTW, fixer does not run for empty file
https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/1.9/Symfony/CS/Fixer.php#L169